variables.json

```diff
@@ -628,6 +628,7 @@
       "Islandora_LOUD1",
       "Islandora_MCAV1",
       "Islandora_MCBP2",
+      "Islandora:MISCE",
```

Comment:
@ebtoner should this be an underscore rather than a colon?

Reply:
Yes, I will update in both files. I had just copied over what was emailed us without doublechecking.